hera_cal/delay_filter.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+from hera_cal import io
+from pyuvdata import UVData
+from hera_cal.datacontainer import DataContainer
+from collections import OrderedDict as odict
+from uvtools.dspec import delay_filter
+from copy import deepcopy
+from scipy import constants
+
+class Delay_Filter():
+
+    def __init__(self):
+        '''Class for loading data, performing uvtools.dspec.delay_filter, and writing out data using pyuvdata.
+        To use, run either self.load_data() or self.load_dicts(), then self.run_filter(). If data is loaded with a single 
+        string path or a single UVData object, it can be written to a new file using self.write_filtered_data().
+        '''
+        self.writable = False
+
+
+    def load_data(self, input_data, format='miriad'):
```

Comment:
Can we change "format" to "file_type". format is a python builtin and we use file_type in other places.

Reply:
I'm using filetype (no underscore, which matches abscal and firstcal).